src/main/java/org/mariadb/jdbc/internal/stream/PacketOutputStream.java

```diff
@@ -1,12 +1,17 @@
 package org.mariadb.jdbc.internal.stream;
 
-import java.io.*;
-import java.nio.ByteBuffer;
-import java.nio.ByteOrder;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.Reader;
+import java.io.UnsupportedEncodingException;
 import java.sql.Timestamp;
 import java.util.Calendar;
 import java.util.zip.DeflaterOutputStream;
 
+import org.mariadb.jdbc.internal.util.ByteBuffer;
```

Comment:
Ok i will do it for the next commit.

On Fri, Mar 4, 2016 at 3:25 PM, Mike Jensen <notifications@github.com>
wrote:

> In src/main/java/org/mariadb/jdbc/internal/stream/PacketOutputStream.java
> <https://github.com/MariaDB/mariadb-connector-j/pull/72#discussion_r55035090>
> :
>
> >  import java.sql.Timestamp;
> >  import java.util.Calendar;
> >  import java.util.zip.DeflaterOutputStream;
> >
> > +import org.mariadb.jdbc.internal.util.ByteBuffer;
>
> If you are not going to use a NIO ByteBuffer I think you should at minimum
> rename this class, to avoid confusion. Anyone who is familiar with java
> will be confused by this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MariaDB/mariadb-connector-j/pull/72/files#r55035090>.
>